test/recipes/30-test_evp.t

```diff
@@ -130,6 +130,9 @@ my @defltfiles = qw(
                      evppkey_kdf_scrypt.txt
                      evppkey_kdf_tls1_prf.txt
                      evppkey_rsa.txt
+                     evppkey_mlkem768_keygen.txt
```

Comment:
shouldn't they be done (pushed to defltfiles) only if no_mlkem is not set?

Reply:
Updated: https://github.com/baentsch/openssl/compare/b8e8d90a0a369a69f399885419df3d793006fe3a..3cc382434d4b8f576fc6213e5741b55ffe6ce396

Reply:
Not quite OK: The new variable name is "no_mlkem" (not declared as such). After fixing that, the tests fail for me (both, test_evp and test_evp_extra): Do I need to do some special setup to run them? I used `make TESTS='test_evp_extra test_evp' test`...

Reply:
Oops, there was a couple incorrect things after fixing a rebase. All tests succeed for me when I run `make test`... can you try again with the latest commit?